modules/configuring-gpu-acceleration-on-your-cluster.adoc

```diff
@@ -0,0 +1,62 @@
+:_module-type: PROCEDURE
+
+[id="configuring-gpu-acceleration-openshift-cluster_{context}"]
+= Configuring GPU acceleration on an OpenShift cluster
+
+You must configure GPU acceleration on your {openshift-platform} cluster to support efficient large language model (LLM) inference, such as when using vLLM. vLLM is optimized for NVIDIA GPUs and does not perform efficiently on CPU-only nodes.
```

Comment:
suggest change ",such as when using vLLM."
 to
 ",for example, when you use vLLM."